governance/api.go

```diff
@@ -167,10 +167,8 @@ type VoteList struct {
 func (api *PublicGovernanceAPI) MyVotes() []*VoteList {
 
 	ret := []*VoteList{}
-	api.governance.voteMapLock.RLock()
-	defer api.governance.voteMapLock.RUnlock()
 
-	for k, v := range api.governance.voteMap {
+	for k, v := range api.governance.voteMap.Copy() {
```

Comment:
What's the reason of calling `Copy()` and then copy the each element into the new one for `ret`?
How about holding a lock and then copying the item here? This seems to cause unnecessary double copying.

Reply:
@ehnuje I'm not sure on where the double copying happened. AFAIK, in `range` operation `Copy()` will be executed once. Then this loop iterates for converting a map into a slice which is to be shown to an user. If I have a wrong understanding, please let me know.

Reply:
First Copy = call `voteMap.Copy()`  
Second Copy = create `VoteList` in the loop (line 172~177), copy `k`, `v.Value`, `v.Casted` and `v.Num`
Hence  `k`, `v.Value`, `v.Casted` and `v.Num` seem to be copied twice, doesn't it?